functorch/functorch/_src/aot_autograd.py

```diff
@@ -206,74 +206,62 @@ def new_fn(args):
     return new_fn
 
 
-@contextmanager
-def _disable_jit_autocast():
-    old_jit_autocast_flag = torch._C._jit_set_autocast_mode(False)
-    try:
-        yield
-    finally:
-        torch._C._jit_set_autocast_mode(old_jit_autocast_flag)
-
-
 def aot_dispatch_autograd(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
-    with _disable_jit_autocast():
-        joint_forward_backward = create_joint_forward_backward(flat_fn)
-        # Set input tensors that require grad to leaves
-        with torch.set_grad_enabled(True):
-            out = flat_fn(*flat_args)
-        out = pytree.tree_map(
-            lambda x: x.detach().contiguous() if isinstance(x, Tensor) else x,
-            out,
-        )
-
-        if isinstance(out, (list, tuple)):
-            num_outs = len(out)
-        else:
-            num_outs = 1
+    joint_forward_backward = create_joint_forward_backward(flat_fn)
+    with torch.set_grad_enabled(True):
+        out = flat_fn(*flat_args)
+    out = pytree.tree_map(
+        lambda x: x.detach().contiguous() if isinstance(x, Tensor) else x,
+        out,
+    )
 
-        joint_inputs = (flat_args, out)
-        with torch.set_grad_enabled(True):
-            fx_g = make_fx(joint_forward_backward, aot_config.decompositions)(
-                *joint_inputs
-            )
+    if isinstance(out, (list, tuple)):
+        _num_outs = len(out)
+    else:
+        _num_outs = 1
 
-            if config.use_functionalize:
-                # Functionalize the foward backward graph. First create a
-                # fake fn to make functionalize happy
-                def fake_fn(primals, tangents):
-                    return fx_g(primals, tangents)
+    joint_inputs = (flat_args, out)
+    with torch.set_grad_enabled(True):
+        fx_g = make_fx(joint_forward_backward, aot_config.decompositions)(*joint_inputs)
 
-                fx_g = make_fx(functionalize(fake_fn))(*joint_inputs)
+        if config.use_functionalize:
+            # Functionalize the foward backward graph. First create a
+            # fake fn to make functionalize happy
+            def fake_fn(primals, tangents):
+                return fx_g(primals, tangents)
 
-        if config.debug_joint:
-            print(fx_g.code)
```

Comment:
I don't see a way to debug_joint anymore, is this intentional?

Reply:
oops